### PR TITLE
[Release-1.2.4] azurestack admin tp2 changes

### PR DESCRIPTION
**Pull request by ptsurbeleu-msft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ptsurbeleu-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pavelt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.csproj [hunk 1]
```diff
@@ -88,8 +84,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.6.4\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.0.1\lib\net45\Microsoft.Rest.ClientRuntime.dll</HintPath>
+    <Reference Include="Microsoft.Rest.ClientRuntime">
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.1.8.2\lib\net45\Microsoft.Rest.ClientRuntime.dll</HintPath>
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.csproj]: @ptsurbeleu-msft Why did you downgrade client-runtime?

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Common/CommonOperations.psm1 [hunk 2]
```diff
@@ -692,3 +526,44 @@ function Get-DeploymentStatus
...
+    )
+
+    foreach ($namespace in $Namespaces)
+    {
+        Register-AzureRmResourceProvider -ProviderNamespace $Namespace -Force
+    }
+
+    # TODO: Remove the need for this sleep
+    # BUG 7391118: Expose QuotaSyncState thru Get Admin and Tenant subscription
+    Start-Sleep -Seconds 30
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Common/CommonOperations.psm1]: @ptsurbeleu-msft please change this to Wait-Seconds

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Common/CommonOperations.psm1 [hunk 3]
```diff
@@ -692,3 +526,44 @@ function Get-DeploymentStatus
...
+    )
+
+    foreach ($namespace in $Namespaces)
+    {
+        Unregister-AzureRmResourceProvider -ProviderNamespace $Namespace -Force
+    }
+
+    # TODO: Remove the need for this sleep
+    # BUG 7391118: Expose QuotaSyncState thru Get Admin and Tenant subscription
+    Start-Sleep -Seconds 30
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Common/CommonOperations.psm1]: @ptsurbeleu-msft please change this to Wait-Seconds

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/packages.config [hunk 4]
```diff
@@ -4,17 +4,16 @@
...
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.0.6030.17499-prerelease" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.26.305102204" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.0.1" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="1.8.2" targetFramework="net45" />
```
Comment (hovsepm) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/packages.config]: Why downgraded?

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.csproj (hunk 1 above)
Comment (ptsurbeleu-msft) [inline src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.csproj]: Not sure what was the original reason (if any) for downgrade... will switch to use 2.0.1.

**Comment by hovsepm** [comment]
on-demand job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1008/

**Comment by ptsurbeleu-msft** [comment]
Thanks for kicking it out. What this job run is for?


Best regards,

Pavel Tsurbeleu.

________________________________

**Comment by hovsepm** [comment]
@ptsurbeleu-msft this has all the tests (Check-in job contains just some part of the tests). The execution time takes about an hour.

**Pull request merged**